src/image/components/runtime.tsx

```diff
@@ -30,9 +40,32 @@ export const Runtime: React.FC<IProps> = ({ authoredState, report }) => {
     }
   };
 
+  const handleClick = () => {
+    const { url, highResUrl, credit, caption, creditLink } = authoredState;
+    const size = imageSize.current?.width && imageSize.current?.height
+                  ? { width: imageSize.current.width, height: imageSize.current.height }
+                  : undefined;
+    const allowUpscale = authoredState.scaling === "fitWidth";
+    const modalImageUrl = highResUrl ? highResUrl : url;
+    const title = `<strong>${caption || ""}</strong> <em>${credit || ""}</em> ${creditLink && ReactDOMServer.renderToString(getCreditLink())}`;
+    modalImageUrl && showModal({ uuid: "image", type: "lightbox", url: modalImageUrl,
```

Comment:
Forgot to check this earlier -- you need to generate a real UUID here.

Reply:
Thanks - added.